test/common/config/subscription_test_harness.h

```diff
@@ -49,6 +49,10 @@ class SubscriptionTestHarness {
   virtual void deliverConfigUpdate(const std::vector<std::string>& cluster_names,
                                    const std::string& version, bool accept) PURE;
 
+  virtual void deliverConfigUpdate(const std::vector<std::string>& cluster_names,
```

Comment:
Sorry, I think you don't need to define it here. Since the `deliverConfigUpdate` that has code is relevant to HTTP only?